docker-compose-setup/docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+services:
+  coatrack-postgres:
+    image: postgres:9.4.26
+    container_name: coatrack-postgres
+    volumes:
+      - coatrack-postgres:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: password1234
+  coatrack-config-server:
+    image: coatrack/coatrack-config-server:2.0.0-SNAPSHOT
+    container_name: coatrack-config-server
+    depends_on:
+      - coatrack-postgres
+    ports:
+      - 8998:8998
+    entrypoint: ["sh", "-c", "java -Djava.security.egd=file:/dev/./urandom \
+      -jar /opt/coatrack/lib/coatrack-config-server.jar \
+      --spring.datasource.url=jdbc:postgresql://coatrack-postgres/ygg_config_server \
+      --spring.datasource.password=password1234"]
```

Comment:
I am wondering: doesn't the config server require url and credentials to access the git repo?

Reply:
This entrypoint command is to **override** parameters whose default value do not fit our requirements. The git repo parameters are already defined in the application.yml of the config server.